src/store/store.ts

```diff
@@ -94,48 +104,75 @@ export const useStore = create<State & Actions>()(
               buildingOutputs: newBuildingOutputs,
             };
           }),
+        // add new saved location
+        saveLocation: (t, x, y) =>
+          set((state) => {
+            const newLocationType = { ...state.locations };
+            newLocationType[t] = { positionX: x, positionY: y };
+            return {
+              locations: newLocationType,
+            };
+          }),
+        // remove saved location
+        removeLocation: (t) =>
+          set((state) => {
+            const newLocationType = { ...state.locations };
+            delete newLocationType[t];
+            return {
+              locations: newLocationType,
+            };
+          }),
```

Comment:
please always use meaningful variable names :) x and y is fine, but t doesn't say anything. You can call it locationType or something like that

Reply:
also, the two comments are not needed, as the function names are self explanatory